lib/files/grid_files_screen.dart

```diff
@@ -113,17 +143,15 @@ class GridFilesScreenState extends State<GridFilesScreen> {
           .toList();
 
       final List<OrionApiItem> items = [...dirs, ...files];
-      if (items.isNotEmpty) {
-        _parentPath = items.first.parentPath;
-      }
+      if (items.isNotEmpty) {}
 
-      if (kDebugMode) {
+      /*if (kDebugMode) {
```

Comment:
we should probably scrap this? Seems unlikely we'll put it back in for code that'll actually get rolled out. 

If the messaging seems like it could be useful, put it in a _logger call at debug level or something

Reply:
Resolved by [b69cbe9](https://github.com/TheContrappostoShop/Orion/pull/14/commits/b69cbe9f1d3eb97f6bd2364059a75745aacde57b)